

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m08:21:11  Running with dbt=1.7.18
[0m08:21:11  Registered adapter: firebolt=1.6.4
[0m08:21:11  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m08:21:11  
[0m08:21:12  Concurrency: 1 threads (target='default')
[0m08:21:12  
[0m08:21:12  1 of 1 START seed file seed .................................................... [RUN]
[0m08:21:13  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 0.82s]
[0m08:21:13  
[0m08:21:13  Finished running 1 seed in 0 hours 0 minutes and 1.42 seconds (1.42s).
[0m08:21:13  
[0m08:21:13  [32mCompleted successfully[0m
[0m08:21:13  
[0m08:21:13  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m08:21:13  Running with dbt=1.7.18
[0m08:21:13  Registered adapter: firebolt=1.6.4
[0m08:21:13  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m08:21:13  
[0m08:21:13  Concurrency: 1 threads (target='default')
[0m08:21:13  
[0m08:21:13  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m08:21:14  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 0.77s]
[0m08:21:14  
[0m08:21:14  Finished running 1 incremental model in 0 hours 0 minutes and 1.39 seconds (1.39s).
[0m08:21:14  
[0m08:21:14  [32mCompleted successfully[0m
[0m08:21:14  
[0m08:21:14  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m08:21:14  Running with dbt=1.7.18
[0m08:21:14  Registered adapter: firebolt=1.6.4
[0m08:21:14  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 431 macros, 0 groups, 0 semantic models
[0m08:21:14  
[0m08:21:15  Concurrency: 1 threads (target='default')
[0m08:21:15  
[0m08:21:15  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m08:21:16  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 0.74s]
[0m08:21:16  
[0m08:21:16  Finished running 1 incremental model in 0 hours 0 minutes and 1.35 seconds (1.35s).
[0m08:21:16  
[0m08:21:16  [31mCompleted with 1 error and 0 warnings:[0m
[0m08:21:16  
[0m08:21:16    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "e23f4072-f31e-40da-a7eb-a93d6198e566",
      "request_id": "d2c8f863-b13e-4203-b752-4ab5d67d864f"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m08:21:16  
[0m08:21:16  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
